### PR TITLE
Issues related enslint-configuration

### PR DESCRIPTION
**Pull request by komalsingh54**
✔︎ I've tested in all browsers (IE11, Firefox, Safari, Chrome, and iOS)

### What should this PR do?
resolve conflicts

### What is the acceptance criteria?
1. List

[If there are UI changes, include a before and after screenshot.]

### CODE REVIEW

File: src/scripts/components/Alert.jsx [hunk 1]
```diff
@@ -35,12 +35,12 @@ const Alert = (props) => {
...
     <div className={classes}>
       {renderClose()}
-      {title &&
-        <div className="alert__title">
-          {renderTitleIcon()}
-          <div className="alert__title__text">{title}</div>
-        </div>
-      }
+      {title && (
+      <div className="alert__title">
```
Comment (benbruning) [inline src/scripts/components/Alert.jsx]: This block should be indented.

File: src/scripts/components/Checkbox.jsx [hunk 2]
```diff
@@ -61,20 +61,20 @@ class Checkbox extends React.Component {
...
-            {label}
-          </label>
-        }
-        {((checked) && children) &&
-          <div className="form__block-group__meta">
-            {children}
-          </div>
-        }
+        {label && (
+        <label // eslint-disable-line jsx-a11y/label-has-for
```
Comment (benbruning) [inline src/scripts/components/Checkbox.jsx]: Should be indented.

File: src/scripts/components/Checkbox.jsx [hunk 3]
```diff
@@ -61,20 +61,20 @@ class Checkbox extends React.Component {
...
+          className="rhinobox__label"
+          htmlFor={this.id}
+          ref={(checkboxLabel) => { this.checkboxLabel = checkboxLabel; }}
+        >
+          {label}
+        </label>
+        )}
+        {((checked) && children) && (
+        <div className="form__block-group__meta">
```
Comment (benbruning) [inline src/scripts/components/Checkbox.jsx]: Indent.

File: src/scripts/components/DropdownMultiSelect.jsx [hunk 4]
```diff
@@ -218,12 +218,13 @@ class DropdownMultiSelect extends React.Component {
         </DropdownWrapper>
         <FormValidationMessage validationMessage={validationMessage} />
         <FormExplanationMessage explanationMessage={explanationMessage} />
-        {activeKeys.length ?
+        {activeKeys.length ? (
```
Comment (benbruning) [inline src/scripts/components/DropdownMultiSelect.jsx]: Could this just be &&?

File: src/scripts/components/FormLabel.jsx [hunk 5]
```diff
@@ -7,13 +7,14 @@ const FormLabel = (props) => {
   const classes = cx(className);
 
   return (
-    children ?
+    children ? (
```
Comment (benbruning) [inline src/scripts/components/FormLabel.jsx]: &&?

File: src/scripts/components/Input.jsx [hunk 6]
```diff
@@ -157,11 +158,11 @@ class Input extends React.Component {
             readOnly={readOnly}
             ref={ref => (this.input = ref)}
           />
-          {input && clear &&
-            <Button reset className="form__clear__button" onClick={this.handleClear} >
-              <Icon icon="close" />
-            </Button>
-          }
+          {input && clear && (
```
Comment (benbruning) [inline src/scripts/components/Input.jsx]: Ident.

File: src/scripts/components/Input.jsx [hunk 7]
```diff
@@ -128,11 +129,11 @@ class Input extends React.Component {
               readOnly={readOnly}
               htmlRef={ref => (this.input = ref)}
             />
-            {input && clear &&
-              <Button reset className="form__clear__button" onClick={this.handleClear} >
-                <Icon icon="close" />
-              </Button>
-            }
+            {input && clear && (
+            <Button reset className="form__clear__button" onClick={this.handleClear}>
```
Comment (benbruning) [inline src/scripts/components/Input.jsx]: Indent.

File: src/scripts/components/ResourceIntro.jsx [hunk 8]
```diff
@@ -50,14 +50,14 @@ class ResourceIntro extends React.Component {
...
-            {checkbox &&
-              <Checkbox
-                label={checkbox.label}
-                name={checkbox.name}
-                isChecked={checkbox.isChecked}
-                onClick={this.handleCheckboxClick}
-              />
-            }
+            {checkbox && (
+            <Checkbox
```
Comment (benbruning) [inline src/scripts/components/ResourceIntro.jsx]: Indent.

File: src/scripts/components/ResourceIntro.jsx [hunk 9]
```diff
@@ -34,14 +34,14 @@ class ResourceIntro extends React.Component {
...
```
Comment (benbruning) [inline src/scripts/components/ResourceIntro.jsx]: Indent.

File: src/scripts/components/Textarea.jsx [hunk 10]
```diff
@@ -74,17 +74,17 @@ class Textarea extends React.Component {
...
-              <div className="form__control-footer__left">
-                <FormValidationMessage validationMessage={validationMessage} />
-                <FormExplanationMessage explanationMessage={explanationMessage} />
-              </div>
-            }
-            {showCharacterCount()}
+        {(validationMessage || explanationMessage || showCharacterCount()) && (
+        <div className="form__control-footer">
+          {(validationMessage || explanationMessage) && (
+          <div className="form__control-footer__left">
```
Comment (benbruning) [inline src/scripts/components/Textarea.jsx]: Indent.

File: src/scripts/components/Checkbox.jsx (hunk 2 above)
Comment (juanfabrega) [reply src/scripts/components/Checkbox.jsx]: Doned

File: src/scripts/components/Checkbox.jsx (hunk 3 above)
Comment (juanfabrega) [reply src/scripts/components/Checkbox.jsx]: Done

File: src/scripts/components/Input.jsx (hunk 6 above)
Comment (juanfabrega) [reply src/scripts/components/Input.jsx]: done

File: src/scripts/components/ResourceIntro.jsx (hunk 8 above)
Comment (juanfabrega) [reply src/scripts/components/ResourceIntro.jsx]: Done

File: src/scripts/components/Textarea.jsx (hunk 10 above)
Comment (juanfabrega) [reply src/scripts/components/Textarea.jsx]: done

File: src/scripts/components/ResourceIntro.jsx (hunk 9 above)
Comment (juanfabrega) [reply src/scripts/components/ResourceIntro.jsx]: done

**Comment by komalsingh54** [comment]
@juanfabrega i just `git merge master` in current branch

**Pull request merged**